src/app/Common/ConsoleBackground/ConsoleBackground.css

```diff
@@ -0,0 +1,4 @@
+.background-img {
```

Comment:
@dpanshug this does not work when you are in production built code. this gets overriden by the selector 

.pf-v5-c-background-image

we need to use the selectors in the components and check the variables associated